### DISCUSSION TITLE
New Vulnerabilities Notification

### DISCUSSION THREAD

**Discussion by rf-azevedo** [DefectDojo/django-DefectDojo#11263] [Q&A] [answered]
Hi all,
I would like to understand if it is possible somehow enable notifications at the vulnerability level.
The use case is the following, I have several engagements where regular scans are performed (Nessus, Nmap, etc.) and I would like to be notified every time a new vulnerability pops up from those scans associated with a particular engagement. Once I have configured several products and some with more than one engagement this would save me from the pain of manually checking each engagement for newly introduced vulnerabilities. 

Thank you in advance.

**Comment by valentijnscholten** [comment, marked as answer]
The `scan_added` notification will fire if an import or reimport resulted in new/updated/closed findings. It will not trigger if the import/reimport did not make any changes to the set of findings already present.

https://docs.defectdojo.com/en/notifications/configure_system_notifs/

**Reply by jWeck-Software** [reply]
https://docs.defectdojo.com/en/notifications/configure_system_notifs/ -> 404

**Reply by mtesauro** [reply]
Docs is getting some serious revamping. That's now here:
https://docs.defectdojo.com/admin/notifications/configure_system_notifs/#main-content
